JamesSedlacek.txt

```diff
@@ -1,3 +1,2 @@
 Hi my name is James Sedlacek.
-3 words that describe me would be: Driven, Dedicated, Disciplined
-My favorite hobby is lifting weights. More specifically, powerlifting. I've competed in powerlifting once. I was in the 148 pound weight class. I set the bench press record for my weight class.
```

Comment:
I would definitely never set a weight lifting record. I've tried lifting, but never stuck with it.